### PR TITLE
Slash color validation [closes #35]

### LINKED ISSUE #35: The /color command accepts arbitrary input
The color command doesn't validate the input being provided, and will use verbatim whatever is provided as coloring information.

ex: setting `/color asdf` will see all messages prepended with `asdf`, where the `a` is interpreted as the color, and `sdf` are inserted as a message prefix.

### PR DESCRIPTION
**Pull request by KuroTsuto**
Adds length and character validation to the `/color` command, and prints appropriate messages when validation fails. Address #35

### CODE REVIEW

File: js/messages.js [hunk 1]
```diff
@@ -77,9 +77,17 @@ MessageList.prototype.handleSlashCommand = function(str) {
 		}
 	} else if (components[0] == 'color') {
 		if (components[1]) {
-			var color = components[1];
-			settings.setColor(color);
-			this.write('Set chat color to "' + color + '". Sample: "' + colorCallback(null, color, 'foo bar baz') + '"');
+			if (components[1].length != 1) {
+				this.write("Please specify a one-character color code.")
+			}
+			else if(!/^[a-z0-5]$/i.test(components[0])) {
```
Comment (seanmakesgames) [inline js/messages.js]: components[1]

File: js/messages.js [hunk 2]
```diff
@@ -77,9 +77,17 @@ MessageList.prototype.handleSlashCommand = function(str) {
 		}
 	} else if (components[0] == 'color') {
 		if (components[1]) {
-			var color = components[1];
-			settings.setColor(color);
-			this.write('Set chat color to "' + color + '". Sample: "' + colorCallback(null, color, 'foo bar baz') + '"');
+			if (components[1].length != 1) {
+				this.write("Please specify a one-character color code.")
```
Comment (seanmakesgames) [inline js/messages.js]: maybe add regex [a-zA-Z0-5] or same instruction text as below

**Comment by seanmakesgames** [comment]
Also merge up so that the other PR's commits are not included.

**Comment by KuroTsuto** [comment]
Whoops - I don't know why this was on my master - will fix 👍

**Comment by KuroTsuto** [comment]
Whoops - I must have bjorked the push - other PR shouldn't have been on master. Will fix

**Comment by KuroTsuto** [comment]
@seanmakesgames I think I've removed the erroneous commits and addressed your suggestions 👍

**Pull request merged**